src/data_gradients/datasets/__init__.py

```diff
@@ -0,0 +1,4 @@
+from data_gradients.datasets.detection import PairedImageLabelDetectionDataset
+from data_gradients.datasets.bdd_dataset import BDDDataset
+
+__all__ = ["PairedImageLabelDetectionDataset", "BDDDataset"]
```

Comment:
call it YoloFormatDetectionDataset

Reply:
Changed in https://github.com/Deci-AI/data-gradients/pull/114/commits/6f40aaf70aa694e3f8a79dd37ece3155abcbc8ec